src/tests/Equinor.ProCoSys.IPO.WebApi.IntegrationTests/Invitations/InvitationsControllerTestsHelper.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System.Net;
+using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+using Equinor.ProCoSys.IPO.Domain.AggregateModels.InvitationAggregate;
+using Newtonsoft.Json;
+
+namespace Equinor.ProCoSys.IPO.WebApi.IntegrationTests.Invitations
+{
+    public static class InvitationsControllerTestsHelper
+    {
+        private const string _route = "Invitations";
```

Comment:
when do you use "_variable" vs "Variable"? I think this should be "Route" if I look at files where you have both styles?

Reply:
as above